modules/ROOT/pages/backup-restore/planning.adoc

```diff
@@ -183,6 +183,9 @@ Backing up a database in a clustered environment is not essentially different fr
 Use `SHOW DATABASE <database>` to learn which servers are hosting the database you want to back up.
 See xref:clustering/monitoring/show-databases-monitoring.adoc#show-databases-monitoring-listing-single[Listing a single database] for more information.
 
+Starting from 2025.09, you can use the `--remote-address-resolution` option to let the DBMS select which servers to use as backup sources.
+See xref:backup-restore/online-backup.adoc#_using_remote_address_resolution[] for more details.
+
```

Comment:
I think it would be good to add info about the new option also on this page, where we talk about some general concepts and planning of backup and restore operations.